qownnotes/snapcraft.yaml

```diff
@@ -10,7 +10,7 @@ description: |
 
 apps:
   qownnotes:
-    command: desktop-launch usr/bin/QOwnNotes --snap
+    command: desktop-launch ${SNAP}/usr/bin/QOwnNotes --snap
```

Comment:
@popey reported that it didn't work without it, after I added it it worked again. I could reproduce that problem locally too...